ci/terraform/oidc/api-gateway-frontend.tf

```diff
@@ -36,6 +36,18 @@ resource "aws_api_gateway_usage_plan_key" "di_auth_frontend_usage_plan_key" {
   usage_plan_id = aws_api_gateway_usage_plan.di_auth_frontend_usage_plan.id
 }
 
+resource "aws_api_gateway_resource" "auth_frontend_wellknown_resource" {
+  rest_api_id = aws_api_gateway_rest_api.di_authentication_frontend_api.id
+  parent_id   = aws_api_gateway_rest_api.di_authentication_frontend_api.root_resource_id
+  path_part   = ".well-known"
+}
+
+resource "aws_api_gateway_resource" "auth_frontend_connect_resource" {
+  rest_api_id = aws_api_gateway_rest_api.di_authentication_frontend_api.id
+  parent_id   = aws_api_gateway_rest_api.di_authentication_frontend_api.root_resource_id
+  path_part   = "connect"
+}
+
```

Comment:
What is this for?

Reply:
I had a problem with the lambda integration in API Gateway which I thought this fixed... On second testing it appears to not be doing anything. Will remove.

Reply:
This is the route for the client registry API in the OIDC API, so definitely shouldn't be needed here!